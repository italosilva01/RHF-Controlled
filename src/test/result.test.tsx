import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";

import Result from "../pages/result";

describe("Page Result", () => {
  test("renders without crashing", () => {
    const { container } = render(<Result />);
    expect(
      screen.getByText("Este é o preço de compra do veículo")
    ).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
