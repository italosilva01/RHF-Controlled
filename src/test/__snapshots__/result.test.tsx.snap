// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Result renders without crashing 1`] = `
<div>
  <div
    class="MuiBox-root css-1r5dyym"
  >
    <h4
      class="MuiTypography-root MuiTypography-h4 css-1kiosdo-MuiTypography-root"
    >
      Tabela Fipe: Preço 
       
       
    </h4>
    <div
      class="MuiBox-root css-1y4qae1"
    >
      <h4
        class="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
        style="font-weight: 900;"
      />
    </div>
    <p
      class="MuiTypography-root MuiTypography-body2 css-emfzs4-MuiTypography-root"
    >
      Este é o preço de compra do veículo
    </p>
  </div>
</div>
`;
